pkg/controller/codes/bulk_issue_test.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package codes_test
+
+import (
+	"context"
+	"net/http"
+	"testing"
+
+	"github.com/google/exposure-notifications-verification-server/internal/project"
+	"github.com/google/exposure-notifications-verification-server/pkg/config"
+	"github.com/google/exposure-notifications-verification-server/pkg/controller"
+	"github.com/google/exposure-notifications-verification-server/pkg/controller/codes"
+	"github.com/google/exposure-notifications-verification-server/pkg/database"
+	"github.com/google/exposure-notifications-verification-server/pkg/rbac"
+	"github.com/google/exposure-notifications-verification-server/pkg/render"
+	"github.com/google/exposure-notifications-verification-server/pkg/sms"
+)
+
+func TestRenderBulkIssue(t *testing.T) {
+	t.Parallel()
+	ctx := context.Background()
+
+	db, _ := testDatabaseInstance.NewDatabase(t, nil)
+	realm := database.NewRealmWithDefaults("Test Realm")
```

Comment:
Should this be a clicky-clicky test? Either way. there's a default realm and user (created by migrations) that you can use to make the test cleaner and more succinct.